tests/unit/hmrc/mtdFraudPrevention.test.js

```diff
@@ -58,6 +59,7 @@ describe("FraudPreventionHeaders", () => {
     );
     expect(headers.get("Gov-Client-Browser-Do-Not-Track")).toBe("true");
     expect(headers.get("Gov-Client-Local-IPs")).toBe("127.0.0.1,127.0.0.2");
+    expect(headers.get("Gov-Client-Device-ID")).not.toBe(generateClientDeviceID());
```

Comment:
Same comment as before. These are more integration tests, so much uuid and make sure it is returning the exact value of header you expect